CMakeLists.txt

```diff
@@ -388,12 +388,16 @@ if(NOT CMAKE_SKIP_INSTALL_RULES)
 
   function(mold_install_relative_symlink OLD NEW)
     install(CODE "
-      get_filename_component(PREFIX_ABS \${CMAKE_INSTALL_PREFIX} ABSOLUTE)
+      if(\"\${CMAKE_INSTALL_PREFIX}\" STREQUAL \"\")
+        set(PREFIX_ABS \"/\")
+      else()
+        get_filename_component(PREFIX_ABS \${CMAKE_INSTALL_PREFIX} ABSOLUTE)
+      endif()
```

Comment:
I wonder if

```
get_filename_component(PREFIX_ABS \${CMAKE_INSTALL_PREFIX}/ ABSOLUTE)
```

works instead (notice the trailing `/` at the end of `CMAKE_INSTALL_PREFIX`).

Reply:
Thanks! Indeed it's working, and simpler.